src/randoop/reflection/DefaultReflectionPredicate.java

```diff
@@ -58,6 +57,18 @@ public boolean test(Class<?> c) {
     return visibility.isVisible (c);
   }
 
+  /**
+   * {@inheritDoc}
+   * Does checks for the following cases:
+   * <ul>
+   * <li>Main methods
```

Comment:
Thanks for documenting this case.  Is this a good policy?  Is it a problem to call main?